db/seeds.rb

```diff
@@ -321,18 +321,33 @@ def setup_fake_health_data
 def setup_hmis_admin_access
   return unless ENV['HMIS_HOSTNAME'].present?
 
-  hmis_ds = GrdaWarehouse::DataSource.source.where.not(hmis: nil).first_or_create do |ds|
-    ds.hmis = ENV['HMIS_HOSTNAME']
+  # Create HMIS Administrator role
+  hmis_admin_role = Hmis::Role.where(can_administer_hmis: true).first_or_create do |role|
+    role.name = 'HMIS Administrator'
+  end
+
+  # Create HMIS Data Source
+  hmis_ds = GrdaWarehouse::DataSource.source.where(hmis: ENV['HMIS_HOSTNAME']).first_or_create do |ds|
     ds.name = 'HMIS Data Source'
   end
-  return if Hmis::Role.where(can_administer_hmis: true).exists?
 
-  role = Hmis::Role.create(name: 'HMIS Administrator', can_administer_hmis: true)
+  # Dev only: give a user HMIS Admin access by setting up a basic Access Control List
+  return unless Rails.env.development?
```

Comment:
For non-dev environments, the rest of this should would need to be run manually. We shouldn't be randomly choosing which User is the HMIS admin.